airflow/settings.py

```diff
@@ -666,3 +666,7 @@ def initialize():
     "y",
     "1",
 }
+
+IS_SCARF_ANALYTICS_ENABLED = conf.getboolean("scarf_analytics", "enabled", fallback=True) and (
```

Comment:
No coverage on this either.

Reply:
Since this is a variable like other vars that gets initialized as soon as "airflow" is imported, this is difficult to test. I would need to convert it to a function -- but I think this is trivial.

Reply:
added in https://github.com/apache/airflow/pull/39510/commits/33ceb8cd05dc5832f0602870b01ae5d810c2cefd